<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>10</y>
     <width>541</width>
     <height>471</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>end_joke.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="mainreturnpushButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Intro Bold</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(174, 234, 160, 255), stop:1 rgba(0, 225, 243, 255));
     border-style: outset;
     border-width: 2px;
     border-radius: 30px;
     border-color: beige;
     min-width: 10em;
     color: white;
 padding: 6px;</string>
   </property>
   <property name="text">
    <string>Вернуться на главную</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cipherreturnpushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Intro Bold</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(174, 234, 160, 255), stop:1 rgba(0, 225, 243, 255));
     border-style: outset;
     border-width: 2px;
     border-radius: 30px;
     border-color: beige;
     min-width: 10em;
     color: white;
 padding: 6px;</string>
   </property>
   <property name="text">
    <string>Опробовать еще раз</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
